### ISSUE TITLE
XAML Controls Gallery : Narrator does not announce about the state of "Account", Document options" upon activating the button in Navigation View.

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/microsoft-ui-xaml#2599]
Copied from internal [issue](https://microsoft.visualstudio.com/OS/_workitems/edit/26617896?src=WorkItemMention&src-action=artifact_link).

**Pre-requisite:** 
Install XAML CONTROLS Gallery from Store
Turn on Narrator.
Set narrator to item mode.

**Repro steps :**
Launch XAML Control Gallery app.
Activate "NavigationView" page from All Controls Page.
Navigate to "Hierarchical Navigation View ->Accounts and activate it.
Observe the narrator description .

**Expected result :**
Narrator should announce about the state of "Account", Document options" upon activating the button in Navigation View as expanded.

**Actual result :** 
Narrator does not announce about the state of "Account", Document options" upon activating the button in Navigation View. 

![Xaml_narrator](https://user-images.githubusercontent.com/7758786/83698897-73e1d680-a5b7-11ea-993b-7d56f4ddce08.png)

**Comment by StephenLPeters** [comment]
@ojhad FYI

**Comment by chingucoding** [comment]
Can I look into this issue @ojhad @StephenLPeters @ranjeshj ?

**Comment by ojhad** [comment]
> Can I look into this issue @ojhad @StephenLPeters @ranjeshj ?

Yes, that would be great!
